packages/devextreme/js/__internal/ui/m_multi_view.ts

```diff
@@ -110,6 +110,12 @@ const MultiView = CollectionWidget.inherit({
       index -= count;
     }
 
+    const step = this._swipeDirection > 0 ? -1 : 1;
+
+    while (!this._isItemVisible(index)) {
+      index = (index + step + count) % count;
```

Comment:
This operation seems to be excess to me, no?
```suggestion
      index = (index + step) % count;
```

Reply:
[eb9495e](https://github.com/DevExpress/DevExtreme/pull/28128/commits/eb9495e5a1192db390e67834c0ed4c2764c2b9d4)